admin/listPurchases/listView.html

```diff
@@ -267,6 +267,39 @@
     `;
 }
 
+function markOrderDispatched(id){    
+    let dispatchButton = document.querySelector(".more-info-" + id + " .mark-as-sent-btn");
+    dispatchButton.classList.add("isClicked");
+    requestMarkOrderDispatched(id, (error) => {
+        if(error) {
+            document.write(error.response);
+            return;
+        }
+        entries.find(entry => entry.id == id).status = "dispatched";
+        options.find(o => o.cls == "option-new-orders").el.click();
+    })
+}
+
+function requestMarkOrderDispatched(id, callback){
+    let req = new XMLHttpRequest();
+    req.open("GET", `/admin/orders/${id}/mark-dispatched`);
+    req.onreadystatechange = function() {
+        if(req.readyState != 4) {
+            return;
+        }
+        if(req.status == 200) {
+            return callback(null);
+        }
+        callback({
+            trace: new Error("Error response from server on request"),
+            status: req.statusText,
+            responseType: req.responseType,
+            response: req.response
+        });
+    };
+    req.send();
```

Comment:
`XMLHttpRequest` is deprecated. Use `fetch` instead: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API